tests/Cookie/CookieJarTest.php

```diff
@@ -37,6 +37,11 @@ public function testCreatesFromArray()
         $this->assertCount(2, $jar);
     }
 
+    public function testEmptyJarIsCountable()
+    {
+        self::assertCount(0, new CookieJar());
```

Comment:
As I understand, you want to check the existence of `count` method. Am I right? But actually you check that `count == 0`. So I'm suggesting you to check implementing of the `\Countable` interface.